modules/common-workflows/pages/workflow-sp-upgrade-webui.adoc

```diff
@@ -0,0 +1,61 @@
+[[workflow-sp-upgrade-via-webui]]
+= Service Pack Upgrade via {webui}
+
+
+
+If you want to migrate {sle} client (??) service pack to a newer version, it can be done either on the command line or via {webui}.
```

Comment:
```suggestion
If you want to migrate the registered {sle} client's service pack to a newer version, it can be done either on the command line or via {webui}.
```